packages/shared/src/modules/contributors/libs/validation-schemas/contributor-get-all.validation-schema.ts

```diff
@@ -0,0 +1,19 @@
+import { z } from "zod";
+
+import {
+	ContributorOrderByKey,
+	ContributorValidationMessage,
+} from "../enums/enums.js";
+import { type ContributorGetAllQueryParameters } from "../types/types.js";
+
+const contributorGetAll: z.ZodType<ContributorGetAllQueryParameters> = z.object(
+	{
```

Comment:
Add the rest of the query params too

Reply:
There is a problem with this. `ContributorGetAllQueryParameters` type consist of fields of different non-string types, but our incoming `query` (which we get in controller) have all fields of type string and this leads to error

Reply:
So It means that all of the `ContributorGetAllQueryParameters` fields should be strings, which is probably not ok

Reply:
![image](https://github.com/user-attachments/assets/18d35cce-9e43-402f-baeb-4f55351afe3b)
![image](https://github.com/user-attachments/assets/b8268e01-d0ea-465f-b00a-fdea0172d8c5)

Reply:
Isn't there type for string which is number in zod?

Reply:
No, there isn't. At least I haven't found it so far ((

Here is the only way to validate query params I found so far
![image](https://github.com/user-attachments/assets/2548936b-29bf-4a0b-96f2-a800fac25ac9)
and I have to remove generic type parameter from ZodType, cause otherwise there is mismatch with types.

Maybe I should separate `ContributorGetAllQueryParameters` into two types -- one with string types for ZodType, controller and api, and second one with actual types (booleans, numbers etc) for service/repository and some files on frontend. But this idea doen't sound good to me.

Reply:
Would be better to separate types, but we have no so much time left, so let's leave it as it is